### DISCUSSION TITLE
LETSUNG Mini-pc (N4000C) + Ubuntu 24.04 LTS

### DISCUSSION THREAD

**Discussion by DanieldJonge** [OpenSurv/OpenSurv#18] [Hardware + Operating system]
Works perfectly, currently 3 camera streams with low cpu usage. Thanks!
Specs: 
CPU: intel N4000C
RAM: 6 GB LPDDR4
Wifi connection
SSD: 128 GB M.2 SSD
Display: 1024x600 7 inch screen

**Comment by SvenVD** [comment]
Nice!
